lib/path.js

```diff
@@ -95,7 +95,10 @@ function normalizeString(path, allowAboveRoot, separator, isPathSeparator) {
           lastSegmentLength = 2;
         }
       } else {
-        res += (res.length > 0 ? separator : '') + path.slice(lastSlash + 1, i);
+        if (res.length > 0)
+          res += `${separator}${path.slice(lastSlash + 1, i)}`;
+        else
+          res = path.slice(lastSlash + 1, i);
```

Comment:
`res` is reassigned here and no longer concatenated. I guess this is done on purpose right?

Reply:
At best it's less work for V8. At worst it's no difference to concatenating with the empty string. This change is more or less a tiny revert of changes made to path.js some time ago, so the use of `=` is from an earlier revision.

Reply:
Thank you for the explanation.